src/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from "@angular/core";
+
+@Component({
+	selector: "ibm-breadcrumb",
+	template: `
+  <nav class="bx--breadcrumb bx--breadcrumb--no-trailing-slash" aria-label="breadcrumb">
+    <ng-content></ng-content>
```

Comment:
`Breadcrumb` should support multiple tiers workflow without developers having to do anything.

![multiple tiers breadcrumb](http://www.carbondesignsystem.com/images/breadcrumb-usage-1.png)

Last item in http://www.carbondesignsystem.com/components/breadcrumb/usage

Reply:
@zvonimirfras the React version doesn't have this built into the `breadcrumb` component. I think it's up to the user of the component to follow this usage pattern. Also we would need to add the overflow component first. This is a component we need so we can work on this and then look at putting together a sample showing this usage pattern. Does that sound OK as a plan?